Extension/c_cpp_properties.schema.json

```diff
@@ -82,6 +82,10 @@
                         "type": "string",
                         "pattern": "^\\d{2}\\.\\d{1}\\.\\d{5}\\.\\d{1}$|^8\\.1$"
                     },
+                    "dotConfig": {
+                        "description": "A path to a .config file created by Kconfig system. Kconfig system generate a file with all the defines, CONFIG options selected, to build a project. Examples of projects that use Kconfig system is Linux Kernel and NuttX RTOS.",
```

Comment:
I don't understand how this ", CONFIG options selected," fits into the sentence -- should the commas be removed and the 1st comma replaced by an "and" or is there another intended meaning?